docs/release-notes.md

```diff
@@ -9,6 +9,27 @@ meta:
 ---
 # Release Notes
 
+## 3.2.0
+
+### Automation Rules UI
+We are happy to introduce The Automation Rules UI! Insights provides over 100 checks for Kubernetes clusters and resources. These checks are increasingly run in a variety of contexts, such as CI/CD, admission control, and in-cluster. Some checks may be more appropriate in specific contexts, require a higher level of severity, or trigger different alert/response mechanisms. Rules allow users to modify existing action items within Insights and customize how they see fit. Please find the Automation Rules under Automation in the navigation bar.
```

Comment:
That last sentence feels very formal. What if we change `Please` to something like `You can`?